catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/converter/impl/FeatureCollectionConverterWfs11.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.v1_1_0.catalog.converter.impl;
+
+import com.thoughtworks.xstream.converters.Converter;
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.WfsFeatureCollection;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.FeatureConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class works in conjunction with XStream to convert a {@link WfsFeatureCollection} to XML
+ * according to the GML 3.1.1 spec. It will also convert respective XML into a {@link
+ * WfsFeatureCollection}.
+ */
+public class FeatureCollectionConverterWfs11 implements Converter {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(FeatureCollectionConverterWfs11.class);
+
+  private static final String FEATURE_MEMBER_ELEMENT = "featureMember";
+
+  private static final String FEATURE_MEMBERS_ELEMENT = "featureMembers";
+
+  private Map<String, FeatureConverter> featureConverterMap = new HashMap<>();
+
+  @Override
+  public boolean canConvert(Class clazz) {
+    if (!WfsFeatureCollection.class.isAssignableFrom(clazz)) {
+      LOGGER.debug("Cannot convert: {}", clazz.getName());
```

Comment:
Why not just return false at this point and return true outside the conditional?  That prevents calling `isAssignableFrom` more than once